demos/projects/PC/linux/README.md

```diff
@@ -1 +1,103 @@
-# Coming Soon
\ No newline at end of file
+# Use Linux for simulate a device using Azure IoT Middleware for FreeRTOS
+
+## Get the middleware
+
+Clone the following repo to download all sample device code, setup scripts, and offline versions of the documentation. 
+
+**If you previously cloned this repo in another sample, you don't need to do it again.**
+
+```bash
+git clone https://github.com/Azure-Samples/iot-middleware-freertos-samples.git
+```
+
+To initialize the repo, run the following command:
+
+```bash
+cd iot-middleware-freertos-samples
+git submodule update --init --recursive --depth 1
+```
+
+
+## Install Prerequisites
+
+* [CMake](https://cmake.org/download/) (Version 3.20 or higher)
+
+* Execute the installation script 
+
+```bash
+sudo ./.github/scripts/install_software.sh
+```
+
+* Execute the Network setup script 
+```bash
+sudo .github/scripts/init_linux_port_vm_network.sh
+```
+
+* To run this sample you can use a device previously created in your IoT Hub or have the Azure IoT Middleware for FreeRTOS provision your device automatically using DPS.
+
+IoT Hub | DPS 
+---------|----------
+Have an [Azure IoT Hub](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal) created | Have an instance of [IoT Hub Device Provisioning Service](https://docs.microsoft.com/en-us/azure/iot-dps/quick-setup-auto-provision#create-a-new-iot-hub-device-provisioning-service)
+Have a [logical device](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal#register-a-new-device-in-the-iot-hub) created in your Azure IoT Hub using your preferred authentication method* | Have an [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/how-to-manage-enrollments#create-a-device-enrollment) created in your instance of DPS using your preferred authentication method*
+
+*While this sample supports SAS keys and Certificates, this guide will refer only to SAS keys. 
+
+## Prepare the simulation
+
+To connect the simulated device to Azure, you'll modify a configuration file for Azure IoT settings, rebuild the image, and run it.
+
+Update the file `demo_config.h` with your configuration values.
+
+```bash
+sudo nano demos/projects/PC/linux/config/demo_config.h
```

Comment:
no need to do sudo

nano demos/projects/PC/linux/config/demo_config.h

Reply:
thanks! old habits!!